fe/fe-core/src/main/java/com/starrocks/server/OlapTableFactory.java

```diff
@@ -388,6 +390,11 @@ public Table createTable(LocalMetastore metastore, Database db, CreateTableStmt
                 }
             }
 
+            boolean hasGin = table.getIndexes().stream().anyMatch(index -> index.getIndexType() == IndexType.GIN);
+            if (hasGin && table.enableReplicatedStorage()) {
+                throw new SemanticException("GIN is only supported in not replicated mode");
```

Comment:
`GIN does not support replicated mode` or `replicated mode does not support GIN`